iree/base/CMakeLists.txt

```diff
@@ -137,43 +152,69 @@ iree_cc_library(
   NAME
     file_io
   SRCS
+    #(marbre): Pulled in as dep defined by platform_trampoline_deps("file_io") in the Bazel world
```

Comment:
Is this a TODO for future work? I haven't seen the "#(username)" syntax before. If it's just a comment, maybe leave off the username?

Reply:
Yeah, you're right. Missed the TODO, but this will be superseded by #227 (with #239 as a first step) quite soon..

Reply:
Let's add the TODOs then. If you're mobile or something, I'm happy to add a commit on top of this to do it (and I'll fix the authorship when it gets imported :smile:)

Reply:
Fixed it myself :wink: